saleor/graphql/decorators.py

```diff
@@ -23,15 +23,30 @@ def wrapper(context, *args, **kwargs):
 
 def permission_required(perm):
```

Comment:
As we expect as list or a single item that will then becoming a list, I think it would be more friendly to directly hint it supports multiple values.
```suggestion
def permission_required(perms: Union[str, Iterable[str]]):
```